corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -176,7 +188,20 @@ def get_choices_for_known_values(self, values):
                 selected_locations, include_self=True
             )
 
-        return [Choice(loc.location_id, loc.display_name) for loc in selected_locations]
+        return self._locations_to_choices(selected_locations)
+
+    def default_value(self, user):
+        """Return only the locations this user can access
+        """
+        location = user.get_sql_location(self.domain)
+        if location:
+            return self._locations_to_choices([location])
```

Comment:
No. I discussed this with @sheelio and he wanted this to just be the location that the user is assigned to.